Makefile

```diff
@@ -231,25 +274,28 @@ all: uncompressed
 
 init:
 	@$(MAKE) clean
+	@$(MAKE) -s -C tools
 	@$(MAKE) decompress
 	@$(MAKE) extract -j $(N_THREADS)
 	@$(MAKE) all -j $(N_THREADS)
 	@$(MAKE) compressed
 
+SF := ___  ___\n/ __||  _|\n\__ \|  _|\n|___/|_|\n
 uncompressed: $(ROM)
 ifneq ($(COMPARE),0)
-	@echo "Calculating Rom Header Checksum..."
-	@$(PYTHON) $(COMPTOOL) -r $(ROM) .
```

Comment:
This shouldn't be removed because it's calculating and fixing the ROM header checksum in case the rom doesn't match, so it can still work on real hardware and most emulators, such checksum is verified by IPL3 at boot time.